admin-notice/admin-notice.php

```diff
@@ -18,5 +20,8 @@
 $admin_notice_controller->add(
 	new Admin_Notice(
 		'WordPress 5.5.2 will be released on Friday, October 30th',
-		[ new Date_Condition( '01-07-2020', '30-10-2020 15:00' ) ]
+		[
+			new Date_Condition( '01-07-2020', '30-10-2020 15:00' ),
```

Comment:
Not related to this PR, but I think we should change the date format. In the US they use `mm-dd-yyyy` instead of `dd-mm-yyyy`, so this format will lead to a human error sooner or later.  I suggest `yyyy-mm-dd`, which is much less confusing (and nicer to computers :D)

Reply:
You are right yyyy-mm-dd is more universal. Both formats are accepted (it is just passed to DateTime constructor), I switched the examples though.

Reply:
Oh, that's right, I hadn't looked at the code. Maybe some phpdoc in the `Date_Condition` constructor would help on that. But whenever possible, if examples are `yyyy-mm-dd` it will be better, thanks! 😊